orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -559,9 +592,12 @@ def parse_outputs_for_onnx_export_and_extract_schema(
         # Deepcopy inputs, since input values may change after model run.
         sample_args_copy, sample_kwargs_copy = deepcopy_model_input(*args, **kwargs)
```

Comment:
should we also control whether `args`, `kwargs` are deep copied or not using the same flag to keep the behavior consistent?

Reply:
I indeed thought about this, while args, kwargs copying is to avoid a module inputs to be modified, not exactly the same as the module copying. Plus the clone of module inputs are not time consuming and memory consuming, so I prefer NOT to do it unless needed. We can revisit later for this. I will follow up as well.